src/client/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -47,6 +47,11 @@ public sealed partial class TokenCache : ITokenCacheInternal
         /// </summary>
         internal bool UsesDefaultSerialization { get; set; } = false;
 
+        /// <summary>
+        /// Static, used by both app and user caches to track approximate token cache size.
+        /// </summary>
+        internal static long CacheSize = 0L;
```

Comment:
Cache size includes user+ app token cache. 2 instances of TokenCache class are created (1 with app cache accessors, 1 with user accessors). So moved this size property up a level so it can be accessed by both.